examples/compose.js

```diff
@@ -0,0 +1,53 @@
+import merge from 'lodash/object/merge';
+import map from 'lodash/collection/map';
+
+const getDescriptorProps = (descriptorName, composables) => {
+  return map(composables, 'compose.' + descriptorName);
+};
+
+const createStamp = descriptor => {
+  return ({ instance } = {}) => {
+    const proto = {};
```

Comment:
Yeah, this is more unfinished stuff... this will eventually look something like this:

```js
  const proto = instance ? {} : methods.proto;
```

Remember we always create a new proto if the instance gets passed in.